versatileimagefield/fields.py

```diff
@@ -142,7 +142,7 @@ def formfield(self, **kwargs):
         # This is a fairly standard way to set up some defaults
         # while letting the caller override them.
         defaults = {'form_class': SizedImageCenterpointClickDjangoAdminField}
-        kwargs.update(defaults)
+        defaults.update(kwargs)
```

Comment:
I've found out that the `SizedImageCenterpointClickDjangoAdminField` is required for the VersatileImageField to work on the admin's forms. In that case probably I should have changed `**defaults` to `**kwargs` in the `super` call, though the comment in that matter is misleading. We want to set/*override* the `form_class` no matter what the user supplied us with?